### PR TITLE
Main branch refactor

### PR DESCRIPTION
**Pull request by ahaynes91**
Refactor of what was already in the main branch and de-duplication of code from the existing PRs. 

**Jira Ticket:** https://affordit.atlassian.net/browse/PRIM-451

### CODE REVIEW

File: Affordit_Integration/Reports/index.html [hunk 1]
```diff
@@ -139,7 +139,7 @@ <h5 class="test-status text-pass">Bill Setting API tesing</h5>
 	<div class="card">
 		<div class="card-header" role="tab">
 			<div class="card-title pl-3">
-				<div class="node">Scenario: Get Billing Setting By Id</div>
+				<div class="node">Scenario: Archive an archived application</div>
```
Comment (afrd-ndilworth) [inline Affordit_Integration/Reports/index.html:142]: Is it already in the Archive? Or going into the Archive?

File: Affordit_Integration/Reports/index.html [hunk 2]
```diff
@@ -151,21 +151,192 @@ <h5 class="test-status text-pass">Bill Setting API tesing</h5>
...
 									<i class="fa fa-check text-white"></i>
 								</span>
-								<span>Given I perform GET operation for GetBillingSettingById</span>
+								<span>Given I perform POST operation for commands/ArchiveApplication</span>
+							</div>
+							<div class="d-flex align-items-center justify-content-start given" title="">
+								<span class="alert-icon pass-bg">
+									<i class="fa fa-check text-white"></i>
+								</span>
+								<span>Given the Application and Entity Id is applications/4902</span>
```
Comment (afrd-ndilworth) [inline Affordit_Integration/Reports/index.html:160]: Will `applications/4902` be replaced with actual ID?

File: Affordit_Integration/Reports/index.html [hunk 3]
```diff
@@ -151,21 +151,192 @@ <h5 class="test-status text-pass">Bill Setting API tesing</h5>
...
+								</span>
+								<span>Then the response status code should be OK</span>
+							</div>
+					</div>
+				</div>
+	</div>
+	<div class="card">
+		<div class="card-header" role="tab">
+			<div class="card-title pl-3">
+				<div class="node">Scenario: Archive an unarchived application</div>
```
Comment (afrd-ndilworth) [inline Affordit_Integration/Reports/index.html:180]: `Archive an unarchived application`. Needs clear description.

**Review by afrd-ndilworth — APPROVED** [review]
Looks Good. 

Adding the gitignore and removing all those generated files was an excellent move. 

I have a few nitpicks with the template, but I assume that will change going forward.

File: Affordit_Integration/Reports/index.html (hunk 1 above)
Comment (ahaynes91) [reply Affordit_Integration/Reports/index.html:142]: Already in the archive.

File: Affordit_Integration/Reports/index.html (hunk 2 above)
Comment (ahaynes91) [reply Affordit_Integration/Reports/index.html:160]: That is the intention once the test tenant is pre-seeded with data.

File: Affordit_Integration/Reports/index.html (hunk 3 above)
Comment (ahaynes91) [reply Affordit_Integration/Reports/index.html:180]: Yup I agree. I just wanted to distinguish between the tests quickly.

**Comment by ahaynes91** [comment]
That "index.html" is the report generated after the test runs. But the suggestions will be changed in the appropriate places when made possible.

**Pull request merged**